github actions running error

The following error occurs when running GitHub Actions on Windows operating system
```
Run CONFORMANCE="D:\a\_actions\modelcontextprotocol\conformance\v0.1.14/dist/index.js"
Running: node D:\a\_actions\modelcontextprotocol\conformance\v0.1.14/dist/index.js server --url http://localhost:8000/mcp --expected-failures ./tests/Conformance/conformance-baseline.yml --suite all --verbose
node:internal/modules/cjs/loader:1386
  throw err;
  ^

Error: Cannot find module 'D:\a\webman-mcp\webman-mcp\a_actionsmodelcontextprotocolconformancev0.1.14\dist\index.js'
    at Function._resolveFilename (node:internal/modules/cjs/loader:1383:15)
    at defaultResolveImpl (node:internal/modules/cjs/loader:1025:19)
    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1030:22)
    at Function._load (node:internal/modules/cjs/loader:1192:37)
    at TracingChannel.traceSync (node:diagnostics_channel:328:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:237:24)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:171:5)
    at node:internal/main/run_main_module:36:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v22.22.0
Error: Process completed with exit code 1.
```

https://github.com/luoyue712/webman-mcp/actions/runs/22035652643/job/63667998162?pr=23

--- a/action.yml
+++ b/action.yml
@@ -42,14 +42,14 @@ runs:
     - name: Build conformance tests
       shell: bash
       run: |
-        cd "${{ github.action_path }}"
+        cd "$GITHUB_ACTION_PATH"
         npm ci
         npm run build
 
     - name: Run conformance tests
       shell: bash
       run: |
-        CONFORMANCE="${{ github.action_path }}/dist/index.js"
+        CONFORMANCE="$GITHUB_ACTION_PATH/dist/index.js"
 
         # Build the command arguments
         ARGS="${{ inputs.mode }}"

